Intro to css values (#44937)
mdn/content: files/en-us/web/css/reference/properties/font-stretch/index.md

font-stretch: unset;
```

### Values

> This property is specified as a {{cssxref("&lt;percentage&gt;")}} value or one of the following keyword values:

Reviewer: ```suggestion
This property is specified as a single value from the following list:
```
